bedrock/mozorg/templates/mozorg/plugincheck.html

```diff
@@ -97,7 +97,7 @@ <h1 class="large">{{_('Check Your Plugins')}}</h1>
       <div class="version-message-container">
         <h2>{{_('Looks like you’re using an older version of Firefox.')}}</h2>
         <div class="desktop download-button-wrapper">
-          {{ download_firefox(simple=true, check_old_fx=true) }}
+          {{ download_firefox(check_old_fx=true) }}
```

Comment:
Note to self: seems `check_old_fx` may be redundant now? The corresponding code no longer seems to do anything, since buttons no longer have subtitles: https://github.com/mozilla/bedrock/blob/master/media/js/base/global.js#L52

This seems to be the only page where this is used. Should we just remove the code?